pkgs/tools/security/maltego/default.nix

```
@@ -0,0 +1,65 @@
+{ lib, stdenvNoCC, fetchzip, makeWrapper, jre, bash, gawk, icoutils, copyDesktopItems, makeDesktopItem }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "maltego";
+  version = "4.3.0";
+
+  src = fetchzip {
+    url = "https://maltego-downloads.s3.us-east-2.amazonaws.com/linux/Maltego.v${version}.linux.zip";
+    sha256 = "1xflh5vyl6dqbzsam3m0larsrgnwgzikqsfmg77y5qy3nyfa15gf";
+  };
+
+  postPatch = ''
+    substituteInPlace bin/maltego \
+      --replace "/bin/bash" "${bash}/bin/bash" \
+      --replace "/usr/bin/awk" "${gawk}/bin/awk"
+  '';
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = pname;
+      desktopName = "Maltego";
+      exec = pname;
+      icon = pname;
+      comment = "An open source intelligence and forensics application";
+      categories = [ "Network" "Security" ];
+      startupNotify = false;
+    })
+  ];
+
+  nativeBuildInputs = [
+    icoutils
+    makeWrapper
+    copyDesktopItems
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share share/icons/hicolor/48x48/apps
```

```suggestion
    mkdir -p $out/share
```

It doesn't seem necessary to create that dir.